creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -669,6 +672,17 @@ class SnapbackSM {
       await utils.timeout(SyncMonitoringRetryDelayMs, false)
     }
 
+    /**
```

Comment:
pls review this logic - lmk if you think this recording approach is correct / sufficient

Reply:
this looks correct, basically it didn't catch up and we didn't exceed the max clock export threshold so it didn't work, just timed out basically

Reply:
exactly, ok cool